src/components/authentication/CreateProfilePage.tsx

```diff
@@ -73,7 +73,7 @@ export default function CreateProfilePage({
         onFormSubmit={() => {
           navigation.reset({
             index: 0,
-            routes: [{ name: "DevelopmentLinks" }],
+            routes: [{ name: "UserSettings" }],
```

Comment:
Should this route the user to the homepage instead of user settings? Or is meant to navigate to user settings so that a user can 'finish' updating their settings?

Reply:
good catch I havent really been testing that route, I can update it now